### PR TITLE
WIP: feat(feat-flag): Add teams page feat flag

### PR DESCRIPTION
**Pull request by priscilawebdev**
Note: We decided to work in a separate branch. This feat flag will only be merged if we decide to continue working on this project after the hackerweek 😉

### CODE REVIEW

**Review by silent1mezzo — APPROVED** [review]
👍

File: src/sentry/conf/server.py [hunk 1]
```diff
@@ -878,6 +878,8 @@ def create_partitioned_queues(name):
     "organizations:org-subdomains": False,
     # Enable access to more advanced (alpha) datascrubbing settings.
     "organizations:datascrubbers-v2": True,
+    # Enable the new teams page (beta)
+    "organizations:teams-page-v2": True,
```
Comment (markstory) [inline src/sentry/conf/server.py:882]: Defaulting to on is risky,  as it will automatically enable in production, on premise and single tenant scenarios.

Comment (priscilawebdev) [reply src/sentry/conf/server.py:882]: Thanks for the tip Mark ;-)

**Comment by priscilawebdev** [comment]
closing this for now

**Pull request closed without merging**